terraform/main.tf

```diff
@@ -0,0 +1,175 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.8.0"
+    }
+  }
+
+  required_version = ">= 0.14.9"
+}
+
+provider "aws" {
+  profile = "radius-tracker"
+}
+
+# -----------------------------------------------------------------------------
+# Resources: Lambda Bucket
+# -----------------------------------------------------------------------------
+
+
+resource "aws_s3_bucket" "_" {
+  bucket        = "radius-tracker"
+  force_destroy = false
+
+  tags = {
+    project = "radiustracker"
+  }
+}
+
+resource "aws_s3_bucket_acl" "_" {
+  bucket = aws_s3_bucket._.id
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_versioning" "_" {
+  bucket = aws_s3_bucket._.id
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_policy" "reports" {
+  bucket = aws_s3_bucket._.id
+  policy = <<EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "PublicReadGetObject",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": "s3:GetObject",
+            "Resource": "${aws_s3_bucket._.arn}/reports/*"
+        }
+    ]
+  }
+  EOF
+}
+
+resource "aws_s3_bucket_cors_configuration" "example" {
+  bucket = aws_s3_bucket._.bucket
+  cors_rule {
+    allowed_methods = ["GET"]
+    allowed_origins = ["*"]
```

Comment:
I would prefer to specify a concrete origin. I don't think it's particularly dangerous to have a `*` here, but I feel a specific origin is better as a matter of best practice.